src/modules/UI/components/PasswordRecoveryReminderModal/PasswordRecoveryReminderModalActions.js

```diff
@@ -19,7 +19,7 @@ export const checkPasswordRecovery = () => async (dispatch: Dispatch, getState:
   const passwordRecoveryRemindersShown = settings.passwordRecoveryRemindersShown
   const isPasswordRecoverySetup = !!account.recoveryKey
   if (isPasswordRecoverySetup) return
-  const totalDollars = getTotalFiatAmount(state)
+  const totalDollars = getTotalFiatAmount(state, 'iso:USD')
   for (const level in passwordRecoveryRemindersShown) {
```

Comment:
For the purposes of `passwordRecoveryReminderModal` we want to check against $20 so we denominate that calculation in `iso:USD` explicitly.